internal/archive/extract_test.go

```diff
@@ -74,11 +96,52 @@ func TestFetchAndExtract(t *testing.T) {
 		})
 	}
 
-	t.Run("fetch-failure", func(t *testing.T) {
+	t.Run("unsupported-format", func(t *testing.T) {
 		t.SkipNow()
 	})
+}
 
-	t.Run("unsupported-format", func(t *testing.T) {
-		t.SkipNow()
+func Test_downloadFile(t *testing.T) {
+	srv := httptest.NewServer(nil)
+	defer srv.Close()
+
+	t.Run("timeout", func(t *testing.T) {
+		done, err := downloadFile(&errClient{context.DeadlineExceeded}, "http://my-fake-url", "")
```

Comment:
Weird error coming from test: `vet: internal/archive/extract_test.go:109:43: ErrDeadlineExceeded not declared by package os`.

I wonder if this has anything to do with the version of Go in this project / the one you're using locally.

Reply:
Yeah I was running 1.15 locally. Replaced it with `context.DeadlineExceeded`. :+1: